server/plugin/bot.go

```diff
@@ -38,14 +38,14 @@ func (p *Plugin) DMWithAttachments(mattermostUserID string, attachments ...*mode
 func (p *Plugin) dm(mattermostUserID string, post *model.Post) (string, error) {
 	channel, err := p.API.GetDirectChannel(mattermostUserID, p.botUserID)
 	if err != nil {
-		p.API.LogInfo("Couldn't get bot's DM channel", "user_id", mattermostUserID, "error", err.Error())
+		p.API.LogError("Couldn't get bot's DM channel", "userID", mattermostUserID, "Error", err.Error())
```

Comment:
```suggestion
		p.API.LogError("Couldn't get bot's DM channel", "UserID", mattermostUserID, "Error", err.Error())
```
Fix this everywhere.